docs/integrations/cloudwatch/cloudwatch-logs.rst

```diff
@@ -0,0 +1,53 @@
+Send logs to Amazon CloudWatch
```

Comment:
cc: @TibsAtWork  is there a way to get rid off this sentenced case when using proper names?